app/javascript/dashboard/components/layout/sidebarComponents/AccountSelector.vue

```diff
@@ -58,5 +66,43 @@ export default {
       globalConfig: 'globalConfig/get',
     }),
   },
+  methods: {
+    async onChangeAccount(accountId) {
+      const accountUrl = `/app/accounts/${accountId}/dashboard`;
+      window.location.href = accountUrl;
+      await this.$store.dispatch('setActiveAccount', {
+        accountId: accountId,
+      });
+    },
+  },
 };
 </script>
+<style lang="scss" scoped>
+.account-selector--wrap {
+  margin-top: var(--space-normal);
+}
+.account-selector {
+  padding-top: 0;
+  padding-bottom: 0;
+  button {
```

Comment:
Please don't apply styles to the HTML node directly. Use a class name

Reply:
May I know the problem with this approach? We use this approach in a lot of components.

Ex: https://github.com/chatwoot/chatwoot/blob/develop/app/javascript/widget/components/HeaderActions.vue#L111

Reply:
We have done this before by disabling the rule mentioned [here](https://github.com/sds/scss-lint/blob/master/lib/scss_lint/linter/README.md#qualifyingelement). It's not ideal. Let us not keep repeating the same item for the new lines of codes written and adding more tech debt.